src/char/char.h

```diff
@@ -29,6 +29,19 @@ enum {
 	TABLE_GUILD_STORAGE,
 };
 
+typedef enum e_makechar_error {
+	ERROR_MK_SQL = -255,
+	ERROR_INVALID_SLOT,
+	ERROR_MAXLIMIT,
```

Comment:
`MAKECHAR_MAX_LIMIT`. Should include the underscore since it's between the rest of the other values.